### DISCUSSION TITLE
Unable to store key

### DISCUSSION THREAD

**Discussion by GRRedWings** [OpenIDC/mod_auth_openidc#890] [Q&A]
I saw these messages in my log, and I'm wondering if I'm doing something wrong, or if something has changed with Microsoft.  Is it possible to allow for a larger value?

[Thu Jul 21 19:03:29.454826 2022] [oauth2:error] [pid 93:tid 140185836705536] [client 172.1.5.1:51198] oauth2_cache_shm_check_value: could not store value since value size is too large (11106 > 8192)
[Thu Jul 21 19:03:29.454855 2022] [oauth2:error] [pid 93:tid 140185836705536] [client 172.1.5.1:51198] oauth2_cache_set: leave: could NOT store: https://login.microsoftonline.com/86e80137-24cb-4b7f-ad48-3e322110abc6/discovery/v2.0/keys

I did some searching and saw another thread, 
https://github.com/zmartzone/mod_auth_openidc/issues/388

But I'm not setting the way it was described in A.

**Comment by zandbelt** [comment]
this seems to be about mod_oauth2 rather than mod_auth_openidc

**Reply by GRRedWings** [reply]
Sorry about that. I'll look at the documentation for that and see if I can figure it out knowing that.

**Reply by zandbelt** [reply]
those docs are a bit terse but just post the question there and I'll provide input
